### PR TITLE
fix: Microsite Header mobile view and other design update

### PR DESCRIPTION
**Pull request by bluekurinji**

### CODE REVIEW

File: libs/web-components/src/components/microsite-header/MicrositeHeader.svelte [hunk 1]
```diff
@@ -90,28 +86,31 @@
...
+    color: var(--goa-color-text-secondary);
+    padding-left: 1rem;
   }
 
   .service-level {
     font-weight: bold;
-    padding: 0 0.5rem;
-    height: 1.5rem;
+    height: 1.625rem;
+    padding: 0.125rem 0.1875rem 0 0.1875rem;
```
Comment (chrisolsen) [inline libs/web-components/src/components/microsite-header/MicrositeHeader.svelte:111]: Use the flex align-items to center things vertically.

File: libs/web-components/src/components/microsite-header/MicrositeHeader.svelte [hunk 2]
```diff
@@ -129,9 +128,19 @@
   }
 
   .site-text {
-    overflow: hidden;
-    white-space: nowrap;
-    text-overflow: ellipsis;
     color: var(--goa-color-text);
   }
+
+  @media (max-width: 40rem) {
```
Comment (chrisolsen) [inline libs/web-components/src/components/microsite-header/MicrositeHeader.svelte:134]: use px for breakpoints

File: libs/web-components/src/components/microsite-header/MicrositeHeader.svelte [hunk 3]
```diff
@@ -129,9 +128,19 @@
...
+
+    .goa-official-site-header {
+      padding-top: 0.5rem;
```
Comment (chrisolsen) [inline libs/web-components/src/components/microsite-header/MicrositeHeader.svelte:137]: use shorthand css

File: libs/web-components/src/components/microsite-header/MicrositeHeader.svelte (hunk 2 above)
Comment (bluekurinji) [reply libs/web-components/src/components/microsite-header/MicrositeHeader.svelte]: done

File: libs/web-components/src/components/microsite-header/MicrositeHeader.svelte (hunk 1 above)
Comment (bluekurinji) [reply libs/web-components/src/components/microsite-header/MicrositeHeader.svelte:108]: I had to provide the dimensions specified by design as is. And the align-items did not play well

**Review by chrisolsen — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**